### ISSUE TITLE
fix info output for latest Redis 2.6 changes

### ISSUE THREAD

**Issue by jlavallee** [jlavallee/Test-Mock-Redis#9]
Running w/RELEASE_TESTING=1, 05-server.t now fails with errors along the lines of:

```
 t/05-server.t ....... 4/? # testing Redis=HASH(0x7fd7fba97b08)

#   Failed test 'db0 info is correct'
#   at t/05-server.t line 114.
#          got: 'keys=1,expires=0,avg_ttl=0'
#     expected: 'keys=1,expires=0'
```

avg_ttl was added & the Test::Mock::Redis info() output needs to be updated to match

**Comment by valcomm** [comment]
probably can be closed as outputs match in 0.22

**Comment by jlavallee** [comment]
Thanks!
